packages/contractkit/src/governance/proxy.ts

```diff
@@ -63,10 +89,41 @@ export const PROXY_SET_IMPLEMENTATION_SIGNATURE = SET_IMPLEMENTATION_ABI.signatu
 export const PROXY_SET_AND_INITIALIZE_IMPLEMENTATION_SIGNATURE =
   SET_AND_INITIALIZE_IMPLEMENTATION_ABI.signature
 
-export const getInitializeAbiOfImplementation = (proxyContractName: string) => {
-  const implementationABI = require(`../generated/${proxyContractName.replace('Proxy', '')}`)
-    .ABI as AbiItem[]
-  const initializeAbi = implementationABI.find((item) => item.name === 'initialize')
+const findInitializeAbi = (items: AbiItem[]) => items.find((item) => item.name === 'initialize')
+
+const initializeAbiMap = {
```

Comment:
Curious why this change is related to this PR - do we need to remove dynamic requires?

Reply:
Yes. @yorhodes is merging a PR that fixes this and pushed a commit to my branch in the meantime.

Reply:
its merged I think